README

```diff
@@ -229,20 +244,20 @@ Compression Tools:
                 Example of usage:
                 ./scummvm-tools-cli --tool compress_scumm_bun [mode params] [-o outputfile] <infile>
 
-                Default outputfile is inputfile with the extension ".bun"
+                Default outputfile is inputfile with the extension ".bun".
 
-                Please note that FLAC compression will produce larger files
-                than the original, for The Curse of Monkey Island!
+                Note: FLAC compression will produce larger files than the original,
```

Comment:
I think a better style would be
`Note: For "The Curse of Monkey Island", FLAC compression will produce larger files than in the original game.`